### ISSUE TITLE
"Implement Interface" Refactoring Misses Ref Returns

### ISSUE THREAD

**Issue by JasonBock** [dotnet/roslyn#20948]
**Version Used**: 
15.2 (26430.13)

**Steps to Reproduce**:
1) Create an interface that has a ref return:
```
public interface IHaveRefReturn
{
	ref int DoSomething();
}
```
2) Have a class implement that interface
```
public class HaveRefReturn
	: IHaveRefReturn
{ }
```
3) Select the "Implement interface" refactoring.

**Expected Behavior**:
The interface is implemented like this by the refactoring:
```
public class HaveRefReturn
	: IHaveRefReturn
{
	public ref int DoSomething()
	{
		throw new NotImplementedException();
	}
}
```
**Actual Behavior**:
The interface is incorrectly implemented like this:
```
public class HaveRefReturn
	: IHaveRefReturn
{
	public int DoSomething()
	{
		throw new NotImplementedException();
	}
}
```
Essentially the refactoring doesn't "see" that the return is a `ref`, and then the refactoring causes a CS8152 compilation error.

**Comment by Pilchie** [comment]
This was the bug you meant to respond to.

**Comment by CyrusNajmabadi** [comment]
This was fixed in 15.3:

![image](https://user-images.githubusercontent.com/4564579/28338502-058f5e5e-6bbe-11e7-8427-359c621b88b4.png)
